assets/pages/new-contributions-landing/contributionsLandingActions.js

```diff
@@ -19,9 +20,15 @@ const updateContributionType = (contributionType: Contrib): Action =>
 const updatePaymentMethod = (paymentMethod: PaymentMethod): Action =>
   ({ type: 'UPDATE_PAYMENT_METHOD', paymentMethod });
 
-const selectAmount = (amount: Amount): Action => ({ type: 'SELECT_AMOUNT', amount });
+const selectAmount = (amount: Amount, contributionType: Contrib, index: number): Action =>
```

Comment:
as discussed - I think using the index to identify the selected amount is a bit confusing and brittle (e.g. it breaks if the "other amount" button moves). I'd go for something like a type like `Amount | 'other'` and then match on that to select the button, if possible

Reply:
Done ✨